packages/ember-views/lib/system/event_dispatcher.js

```diff
@@ -191,16 +189,18 @@ export default EmberObject.extend({
     @param {Element} rootElement
     @param {String} event the browser-originated event to listen to
     @param {String} eventName the name of the method to call on the view
-    @param {Object} viewRegistry
   */
-  setupHandler(rootElement, event, eventName, viewRegistry) {
+  setupHandler(rootElement, event, eventName) {
     let self = this;
 
+    let owner = getOwner(this);
+    let viewRegistry = owner && owner.lookup('-view-registry:main') || fallbackViewRegistry;
```

Comment:
I've brought back that original change in this PR: https://github.com/emberjs/ember.js/pull/14968